src/app/features/genes/components/gene-evidence-proteomics/gene-evidence-proteomics.component.ts

```diff
@@ -74,10 +78,77 @@ export class GeneEvidenceProteomicsComponent {
       this.selectedUniProtId = this.uniProtIds[0];
     }
 
+    this.initSRM();
+
     this.initLFQ();
     this.initTMT();
   }
 
+  initSRM() {
+    this.geneService.getDistribution().subscribe((data: any) => {
+      const distribution = data.proteomics_SRM;
+
+      const differentialExpression =
+        this._gene?.proteomics_SRM?.filter((item: any) => {
+          return item.uniprotid === this.selectedUniProtId;
+        }) || [];
+
+      const SRMData: any = [];
+
+      differentialExpression.forEach((item: any) => {
+        const data: any = distribution.find((d: any) => {
+          return d.tissue === item.tissue;
+        });
+
+        if (data) {
+          const yAxisMin = item.log2_fc < data.min ? item.log2_fc : data.min;
+          const yAxisMax = item.log2_fc > data.max ? item.log2_fc : data.max;
+
+          if (this.SRMYAxisMin == undefined || yAxisMin < this.SRMYAxisMin) {
+            this.SRMYAxisMin = yAxisMin;
+          }
+
+          if (this.SRMYAxisMax == undefined || yAxisMax > this.SRMYAxisMax) {
+            this.SRMYAxisMax = yAxisMax;
+          }
+
+          SRMData.push({
+            key: data.tissue,
+            value: [data.min, data.median, data.max],
+            circle: {
+              value: item.log2_fc,
+              tooltip:
+                (item.hgnc_symbol || item.ensembl_gene_id) +
+                ' is ' +
+                (item.cor_pval <= 0.05 ? ' ' : 'not ') +
```

Comment:
If the p-value is less than 0.05, should an empty string be used so there isn't an extra space in the sentence?

Reply:
Nice find.  I will remove it and refactor this as it is used in multiiple places.